### ISSUE TITLE
Retrospectively add note (and change note date)

### ISSUE THREAD

**Issue by ronaldsuwandi** [monicahq/monica#1609]
Currently when you add new note it will always use today's date as the date which can be annoying for new user who wants to add notes retrospectively from few months ago.

Also ability to change note date would be great

**Comment by LukasRos** [comment]
+1 I would like to see this feature, too.

**Comment by foldedpaper** [comment]
I agree, sometimes I use notes to log an activity or event that can't easily or quickly be categorized into existing categories, and find myself adding the note some days after the event.
